File: ruby_small_problems/1.rb
# generate random number between 20 and 200
# print string with number interpolated

def print_age(name='Teddy')
  puts "#{name} is #{[*20..200].sample} years old!"
  puts "#{name} is #{rand(20..200)} years old!"
end

print_age
print_age('Bertha')

File: 210/exercises/small_problems/easy_3/1.js
function getRandomArbitrary(min, max) {
  return Math.random() * (max - min) + min;
}

let age;
let i;
for (i = 0; i <= 50; i += 1) {
  age = Math.floor(getRandomArbitrary(20, 201));
  console.log(`Teddy is ${age} years old!`);
}

// 20..100
// random => 0...1 => 0...80
